src/Ship.java

```diff
@@ -12,11 +12,15 @@ public Ship(int life) {
 	// in the observer pattern
 	//
 	public void update() throws Exception {
-
+		this.hit();
 	}
 
 	public boolean isSunk() {
```

Comment:
the sunk property  is intended to be a cache to additional logic from executing (performance)